Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/VmServiceWrapper.java

```diff
@@ -369,16 +366,13 @@ public void received(final Stack vmStack) {
         ApplicationManager.getApplication().executeOnPooledThread(() -> {
           InstanceRef exceptionToAddToFrame = exception;
 
-          ElementList<Frame> elementList;
-          if (RENDER_CAUSAL_FRAMES) {
+          ElementList<Frame> elementList = vmStack.getAwaiterFrames();
```

Comment:
With the VM protocol, do we check that the user is at some version of the SDK, or just assume that they are?